inst/tws/tests/testthat/test-shinytest2.R

```diff
@@ -0,0 +1,24 @@
+library(shinytest2)
+
+test_that("{shinytest2} recording: tws_screenshot", {
+  app <- AppDriver$new(
+    variant = platform_variant(), name = "tws_screenshot",
+    height = 823, width = 1459
+  )
+  app$expect_screenshot()
```

Comment:
I would not recommend screenshot testing. In my experience it's too finnicky and has more cost than benefit.

For example, when I ran the tests the screenshots were flagged as changed. I'm on the same `platform_variant()` too. Looking at the images, it was the tiniest difference, something a human may not even if it wasn't highlighted.

Reply:
So this cannot happened. I though it is too good to be true, to have an image identical up to every pixel